frontend/src/scenes/trends/trendsLogic.ts

```diff
@@ -137,7 +137,7 @@ export const trendsLogic = kea<trendsLogicType<FilterType, ActionType, TrendPeop
 
     loaders: ({ values, props }) => ({
         results: {
-            __default: [],
+            __default: [] as TrendResult[],
```

Comment:
I think you need to pass TrendResult to kea as well - check the types file for if it has any errors.